trufflehog.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: trufflehog
+  version: 3.88.7
+  epoch: 0
+  description: Find, verify, and analyze leaked credentials
+  dependencies:
+    runtime:
+      - bash
+      - binutils
+      - ca-certificates
+      - cpio
+      - git
+      - openssh-client
+      - rpm2cpio
+  copyright:
+    - license: AGPL-3.0-only
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/trufflesecurity/trufflehog
+      tag: v${{package.version}}
+      expected-commit: f2dc96e8548d5fab0f6ffb3d47b9918b49fdde0a
+
+  - uses: go/build
+    with:
+      packages: ./main.go
+      ldflags: -X github.com/trufflesecurity/trufflehog/v3/pkg/version.BuildVersion=${{package.version}}
+      output: trufflehog
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/etc
+      cp entrypoint.sh ${{targets.contextdir}}/etc/entrypoint.sh
+      chmod +x ${{targets.contextdir}}/etc/entrypoint.sh
+
+update:
+  enabled: true
+  github:
+    identifier: trufflesecurity/trufflehog
+    use-tag: true
+    strip-prefix: v
+
+test:
+  pipeline:
+    - name: version and help tests
+      runs: |
+        trufflehog --version
+        trufflehog --help
+    - name: trufflehog scans
+      runs: |
```

Comment:
Can we instead grep the real secrets here to ensure it actually works? "token" keyword does seem a bit open the failure and possible false-positives.

Reply:
switched to using actual secret via https://github.com/wolfi-dev/os/pull/42197/commits/65986bafa3ca41c232533cc9ab702845a07881a4